src/lib/core/style/_menu-common.scss

```diff
@@ -17,9 +17,6 @@ $mat-menu-icon-margin: 16px !default;
 @mixin mat-menu-base($default-elevation) {
   @include mat-overridable-elevation($default-elevation);
 
-  // Prevents the content from repainting on scroll.
```

Comment:
I'm also open to keeping these and scoping them to something like `&:not([dir='rtl'])`, but it seems like it could end up causing some inconsistencies.